Update cv (#4)
aguspesce/cv: cover-letter.tex

\documentclass[a4paper,10pt]{letter}
\usepackage{hyperref}
\usepackage[a4paper]{geometry}
\geometry{top=2.5cm, bottom=2.5cm}

\date{February 25, 2021}

\signature{Agustina Pesce}

% \address{}

\begin{document}

\begin{letter}
{To whom it may concern / Dear Human Resources Team,}
\opening{}

I'm writing to express my interest in working with you in a position like the 

> Data Scientist, Software Engineer or Web developed.

Reviewer: ```suggestion
Data Scientist, Software Engineer or Web Developer.
```

> I am applying in these types of positions due to my scientific and computer 

Reviewer: ```suggestion
I am applying to these positions due to my scientific and computer 
```

> training. 

Reviewer: ```suggestion
knowledge. 
```

I'm a Physicist and a PhD in Geophysics working on scientific research at
\href{https://www.conicet.gov.ar/}{CONICET}.
During my PhD and my postdoc I had the opportunity to learn and improve my
Python programming skills to process, visualize and analyze data. 

> I'm also committed to contribute on open-source software projects to improve

Reviewer: ```suggestion
I'm also committed to contribute to open-source software projects to improve
```

science openness and reproducibility.
In this sense, I've made contributions to the libraries of the
\href{https://fatiando.org}{Fatiando a Terra} project (open-source tools for
geophysics) and lately to \href{https://github.com/ggciag/mandyoc}{Mandyoc}(a 
mantle dynamics simulator).
Moreover, I've started developing 
\href{https://github.com/aguspesce/tapioca}{tapIOca}, a set of Python tools to 
load, visualize and analyze the outputs of Mandyoc.

> To satisfy my curiosity, I also learn about web design and development.

Reviewer: ```suggestion
To satisfy my curiosity, I'm also learning about web design and development.
```

> I collaborated in some project like 

Reviewer: ```suggestion
I collaborated to some project like 
```

\href{https://geolatinas.github.io/}{Geolatinas Coding group} and 
\href{https://www.fatiando.org/}{Fatiando main website}. 

> I also made own design like \href{https://dianaceroallard.github.io/}{Diana 
> personal website}, \href{https://aguspesce.github.io/garabatos/}{Garabatos} and 
> \href{https://aguspesce.github.io/web-cromografica/}{CromoGrafica}. 

Reviewer: ```suggestion
I have also made designs of my own, like \href{https://dianaceroallard.github.io/}{Diana 
personal website}, \href{https://aguspesce.github.io/garabatos/}{Garabatos} and 
\href{https://aguspesce.github.io/web-cromografica/}{CromoGrafica}. 
```

> Having a solid background in Physics provided a strong foundation for building 

Reviewer: ```suggestion
Having a solid background in Physics provided me a strong foundation for building 
```

my knowledge.
It allowed me to satisfy my curiosity and develop myself as a self-taught coder 

> who enjoy learning new tools and improve my computer-science skills.

Reviewer: ```suggestion
who enjoys learning new tools and improve my computer-science skills.
```

On the way I learned how to efficiently solve problems and apply the best
practices for software development.

I would like to thank you for taking the time to review my curriculum and 
consideration. 

> I look forward to hearing about some employment opportunity.

Reviewer: ```suggestion
I look forward to hearing about employment opportunities.
```
